src/components/App.js

```diff
@@ -10,6 +11,17 @@ import ChooseUs from './choose-us/ChooseUs';
 import Footer from './footer/Footer';
 
 function App() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    client
+      .getEntries()
+      .then((response) => {
+        console.log(response);
+      })
+      .catch(console.error);
+  });
```

Comment:
should this just run once on initial App render with an empty dependency array? Does that matter?